src/coreclr/jit/compiler.cpp

```diff
@@ -4095,7 +4095,7 @@ const char* Compiler::compGetTieringName(bool wantShortName) const
     }
     else if (tier1)
     {
-        if (opts.jitFlags->IsSet(JitFlags::JIT_FLAG_OSR))
+        if (opts.IsOSR())
```

Comment:
Seems like we should extend the logic in this method to describe the new "tier1 / instrumented case"